{"uuid":"16c26379-5f12-4070-9b63-e031eefb9210","historyId":"49d32f66c857d2cee2fa3dfe2a12465d","fullName":"com.thetestingacademy.ex03_Selenium_Locators.TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME.test_vwo_login_invalid_login","labels":[{"name":"package","value":"com.thetestingacademy.ex03_Selenium_Locators.TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME"},{"name":"testClass","value":"com.thetestingacademy.ex03_Selenium_Locators.TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME"},{"name":"testMethod","value":"test_vwo_login_invalid_login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"LearningSeleniumATB11x"},{"name":"subSuite","value":"com.thetestingacademy.ex03_Selenium_Locators.TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME"},{"name":"host","value":"Nishant-PC"},{"name":"thread","value":"18824@Nishant-PC.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"owner","value":"Nishant"},{"name":"severity","value":"blocker"}],"links":[],"name":"test_vwo_login_invalid_login","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Response code 500. Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir \nHost info: host: 'NISHANT-PC', ip: '192.168.1.9'\nBuild info: version: '4.32.0', revision: 'd17c8aa950'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'\nDriver info: org.openqa.selenium.edge.EdgeDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: MicrosoftEdge, ms:edgeOptions: {args: [--start-maximized], binary: C:\\Program Files (x86)\\Micr..., extensions: []}}]}]","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: session not created: probably user data directory is already in use, please specify a unique value for --user-data-dir argument, or don't use --user-data-dir \nHost info: host: 'NISHANT-PC', ip: '192.168.1.9'\nBuild info: version: '4.32.0', revision: 'd17c8aa950'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'\nDriver info: org.openqa.selenium.edge.EdgeDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: MicrosoftEdge, ms:edgeOptions: {args: [--start-maximized], binary: C:\\Program Files (x86)\\Micr..., extensions: []}}]}]\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:114)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\r\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:187)\r\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)\r\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:245)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)\r\n\tat org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:110)\r\n\tat org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:60)\r\n\tat org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:56)\r\n\tat org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:48)\r\n\tat com.thetestingacademy.ex03_Selenium_Locators.TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME.test_vwo_login_invalid_login(TestSelenium17_Project_Locators_Selenium_ID_CLASSNAME_NAME.java:25)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"TC#1 - Verify that with invalid login, error message is displayed!","steps":[],"attachments":[],"parameters":[],"start":1747587026155,"stop":1747587029785}